File: app/src/main/java/io/github/romangolovanov/apps/ametro/ui/fragments/SettingsListFragment.java
package io.github.romangolovanov.apps.ametro.ui.fragments;


import android.os.Bundle;
import android.preference.PreferenceFragment;

import io.github.romangolovanov.apps.ametro.R;

public class SettingsListFragment extends PreferenceFragment
{
    @Override
    public void onCreate(final Bundle savedInstanceState)
    {
        super.onCreate(savedInstanceState);
        addPreferencesFromResource(R.xml.settings);
    }
}

File: app/src/main/java/io/github/romangolovanov/apps/ametro/ui/fragments/SettingsListFragment.kt
package io.github.romangolovanov.apps.ametro.ui.fragments

import android.os.Bundle
import androidx.preference.PreferenceFragmentCompat
import io.github.romangolovanov.apps.ametro.R

class SettingsListFragment : PreferenceFragmentCompat() {
    override fun onCreatePreferences(savedInstanceState: Bundle?, rootKey: String?) {
        setPreferencesFromResource(R.xml.settings, rootKey)
    }
}
